pkgs/by-name/pu/pushlog/package.nix

```
@@ -0,0 +1,42 @@
+{
+  lib,
+  fetchFromGitHub,
+  python3Packages,
+}:
+python3Packages.buildPythonApplication rec {
+  pname = "pushlog";
+  version = "0.3.1";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "serpent213";
+    repo = "pushlog";
+    rev = "v${version}";
```

Use tag for tagged releases

```suggestion
    tag = "v${version}";
```